### PR TITLE
fix: password count change from 4 to 6

### LINKED ISSUE #1449: Onboarding: Ensure password is at least 6 characters long
The view has a message it needs to be at least 6 characters long, but we allow 4.
Also, the validation message for too few characters says "4", should be 6 instead

### LINKED ISSUE #1450: Onboarding: don't allow entering the confirmed password when first one is invalid
E.g. if the entered password isn't long enough, it shouldn't be possible to even enter the confirmation

### PR DESCRIPTION
**Pull request by hydr063n**
fixes #1449
fixes  #1450

### CODE REVIEW

File: ui/imports/Utils.qml [hunk 1]
```diff
@@ -197,4 +197,31 @@ QtObject {
             default: return network
         }
     }
+
+    function validate(item, firstPasswordField, repeatPasswordField) {
+        console.log("validate");
```
Comment (jrainville) [inline ui/imports/Utils.qml:202]:
```suggestion
```

File: ui/imports/Utils.qml [hunk 2]
```diff
@@ -197,4 +197,31 @@ QtObject {
             default: return network
         }
     }
+
+    function validate(item, firstPasswordField, repeatPasswordField) {
```
Comment (jrainville) [inline ui/imports/Utils.qml:201]: Can we rename this to `validatePasswords` or something like that

**Review by jrainville — COMMENTED** [review]
Nice. I'd  just change the validate function name so it's more representative, especially since it's in Utils now

File: ui/imports/Utils.qml (hunk 2 above)
Comment (hydr063n) [reply ui/imports/Utils.qml]: Done!

**Review by jrainville — APPROVED** [review]

**Pull request merged**